src/analytics/redux/tracked_events.js

```diff
@@ -19,6 +69,53 @@ function getBlockObject( block ) {
 	return select( 'core/block-editor' ).getBlock( block ) || {};
 }
 
+/**
+ * Track block replacement.
+ *
+ * @param {Array}          originalBlockIds ID(s) or blocks that are being replaced
```

Comment:
Nitpick: Based on the argument's description, I'm not sure if it could be confusing that we mention "ID(s) **or** blocks". I understand we only allow ID(s) **of** blocks. What do you think about updating this? I.e.:
```suggestion
 * @param {Array}          originalBlockIds ID(s) of block(s) that are being replaced
```

Reply:
That's correct, we only allow ID(s) of blocks so I updated it as suggested in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976/commits/b2e4ee34b0b97710deb3dbf3aca02d306da753f8